distribution/src/resources/config-tool/templates/conf/internal-apis.xml.j2

```diff
@@ -20,37 +20,86 @@
 -->
 <internalApis>
     <apis>
-        {% for api in internal_api %}
-        <api name="{{api.api_name}}" protocol="{{api.protocols}}" class="{{api.class}}"/>
-        {% endfor %}
-        <api name="{{management_api.name}}" protocol="{{management_api.protocols}}" class="{{management_api.class}}">
+        <api name="PrometheusApi" protocol="{{prometheus_api.protocols}}" class="org.wso2.micro.integrator.prometheus.publisher.service.MetricAPI">
+            {% if prometheus_api.basic_security_handler.enable == true
+                or prometheus_api.authorization_handler.enable == true or prometheus_api.custom_handlers is defined %}
+            <handlers>
+                {% if prometheus_api.basic_security_handler.enable == true %}
+                <handler class="org.wso2.micro.integrator.management.apis.security.handler.BasicSecurityHandler" name="BasicSecurityHandler"/>
```

Comment:
Shall we have the name as the first attribute and class as second one , just to be consistent with api tags ?. Or is there any specific reason to change it .

Reply:
done